src/components/D3tree.js

```diff
@@ -50,26 +50,35 @@ export default function D3Tree() {
 
     componentNodes.forEach(node => {
       node.addEventListener("mouseover", event => {
-        setNodeId(event.target.id);
+        const nodeData = hierarchyData.find(
+          d => d.data.uuid === event.target.id,
```

Comment:
연산자가 있는 경우에는 `{}` 를 넣어주는게 더 읽기 편한 것 같아요
```suggestion
          d => { d.data.uuid === event.target.id },
```